schematics/shell/index.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  Rule,
+  SchematicContext,
+  Tree,
+  chain,
+  noop,
+  SchematicsException
+} from '@angular-devkit/schematics';
+import {Schema} from './schema';
+import {materialVersion, cdkVersion, angularVersion} from '../utils/lib-versions';
+import {getConfig} from '../utils/devkit-utils/config';
+import {addModuleImportToRootModule} from '../utils/ast';
+import {addHeadLink} from '../utils/html';
+import {addPackageToPackageJson} from '../utils/package';
+
+/**
+ * Scaffolds the basics of a Angular Material application, this includes:
+ *  - Add Packages to package.json
+ *  - Adds pre-built themes to styles.ext
+ *  - Adds Browser Animation to app.momdule
+ */
+export default function(options: Schema): Rule {
```

Comment:
Do you have to use a default export? These are generally strongly discouraged / banned inside Google (and these schematics will someday be used inside Google)

Reply:
@hansl - All of the schematics seem to use default exports (ref: https://github.com/angular/devkit/blob/master/packages/schematics/angular/app-shell/index.ts#L313). Thoughts on @jelbourn comment?